egs/swbd/s5c/local/chain/run_tdnn_7d.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
+
```

Comment:
please add a comparison of the WERs and objective functions vs 6z.  You may be able to use compare_wer.sh.